### PR TITLE
Adds Norsk-mirror pipeline

### PR DESCRIPTION
**Pull request by YoussB**
- Added a pipeline to mirror the Norsk (OSLO) pipeline.

cc @kcmannem

### CODE REVIEW

**Comment by xtreme-sameer-vohra** [comment]
Could we DRY it up by moving the task into a separate task.yml and script.sh ?
Also, update the PR since 3.13 is deprecated

**Comment by vito** [comment]
Is this still a thing we want to do?

**Comment by YoussB** [comment]
it is nice to have that pipeline on the screens. Will update the PR with the latest config,

File: ci/pipelines/norsk-mirror.yml [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+    private_key: ((norsk_deploy_key))
+    uri: git@github.com:pivotal/oslo-scan-status.git
+    paths: [ p-concourse/* ]
+- name: 4.2.X-scan-status
+  type: git
+  source:
+    private_key: ((norsk_deploy_key))
+    uri: git@github.com:pivotal/oslo-scan-status.git
+    paths: [ p-concourse-4.2.x/* ]
+- name: 3.13.X-scan-status
```
Comment (cirocosta) [inline ci/pipelines/norsk-mirror.yml]: we can remove this one 😁

File: ci/pipelines/norsk-mirror.yml [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+  type: git
+  source:
+    private_key: ((norsk_deploy_key))
+    uri: git@github.com:pivotal/oslo-scan-status.git
+    paths: [ p-concourse-3.13.x/* ]
```
Comment (cirocosta) [inline ci/pipelines/norsk-mirror.yml]: can be removed too 😁

File: ci/pipelines/norsk-mirror.yml [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+jobs:
+- name: check-p-concourse-latest
+  plan:
+  - get: latest-scan-status
+    trigger: true
+  - task: check-status-file
+    config:
+      platform: linux
+      image_resource:
+        type: docker-image
```
Comment (cirocosta) [inline ci/pipelines/norsk-mirror.yml]: https://github.com/concourse/registry-image-resource 🙌

**Comment by taylorsilva** [comment]
Moved to new repo: https://github.com/concourse/ci/pull/25

**Pull request closed without merging**